charts/reports/templates/deployment.yaml

```
@@ -66,8 +66,10 @@ spec:
         {{- toYaml . | nindent 8 }}
     {{- end }}
       volumes:
-        - name: html
-{{ toYaml .Values.htmlVolume | indent 10 }}
-        - name: config
-          configMap:
-            name: reports
+      {{- with .Values.htmlVolume  }}
+      - name: html
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      - name: config
+        configMap:
+          name: {{ include "reports.fullname" . }}
```

```suggestion
        {{- with .Values.htmlVolume  }}
        - name: html
          {{- toYaml . | nindent 8 }}
        {{- end }}
        - name: config
          configMap:
            name: {{ include "reports.fullname" . }}
```

nit: trying to diff this PR with an installed `reports` chart shows there is a minor indentation difference.

Non blocking as per https://yaml.org/spec/1.2.2/#61-indentation-spaces (both indent and non indent are OK) but I prefer to avoid bad surprises if you don't mind?

---

LGTM :+1: 
Commited the changes